src/style.css

```diff
@@ -1 +1,70 @@
-/* Styles go here */
+/* stylelint-disable string-quotes */
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap');
+
+body {
+  font-family: 'Roboto', sans-serif;
+  font-size: 12px;
+  margin: 0;
+  background-color: #e5e5e5;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  box-sizing: border-box;
+  width: 100%;
+  background-color: #fff;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.menu {
+  display: flex;
+  height: 100%;
+  margin: 0;
+  padding: 0;
+}
+
+.menu li {
```

Comment:
don't stylize tags and don't use several selectors where you can use just one class